### PR TITLE
Integrations script options and optical flow gazebo sitl support

### PR DESCRIPTION
**Pull request by jgoppert**
* Bash arg handling
* Optical flow rostest script for sitl gazebo
* Groundtruth mavlink stream

### CODE REVIEW

**Comment by LorenzMeier** [comment]
Jenkins test this please.

File: posix-configs/SITL/init/lpe/iris [hunk 1]
```diff
@@ -56,15 +56,16 @@ sensors start
 commander start
 land_detector start multicopter
 navigator start
-attitude_estimator_q start
+#attitude_estimator_q start
+att_mekf start
```
Comment (bkueng) [inline posix-configs/SITL/init/lpe/iris]: What's `att_mekf`?

**Comment by jgoppert** [comment]
Thanks for the catch, was trying to sync another branch and missed that. It is an alternative to att est q that isn't ready yet.

**Comment by LorenzMeier** [comment]
@ChristophTobler Please review and comment.

**Comment by ChristophTobler** [comment]
Ok. I'll have a look at it.

**Comment by LorenzMeier** [comment]
@ChristophTobler Can you review so we can bring this in?

**Comment by LorenzMeier** [comment]
@jgoppert Can you take review feedback into account and rebase? Thanks!

**Comment by jgoppert** [comment]
This failed on SITL  due to the time stamps not being in order somehow for one log. Nothing obvious after plotting. After forcing a sort on px4tools of log data I can analyze the data from the failed log. I have pushed the changes to px4tools and will test again once pypi refreshes.

**Comment by jgoppert** [comment]
@LorenzMeier think this is good to merge.

**Comment by LorenzMeier** [comment]
Nice!

**Pull request merged**